### PR TITLE
Enable log collection from master.

### PR DESCRIPTION
**Pull request by ArtfulCoder**
#5011

Given that we have master components running as pods, enable log collection from the corresponding docker containers.

### CODE REVIEW

**Comment by ArtfulCoder** [comment]
@dchen1107

**Comment by vishh** [comment]
cc @zmerlynn

File: cluster/saltbase/salt/top.sls [hunk 1]
```diff
@@ -56,6 +56,14 @@ base:
     - docker
     - kubelet
 {% endif %}
+{% if pillar.get('enable_node_logging', '').lower() == 'true' %}
+  {% if pillar['logging_destination'] is defined and pillar['logging_destination'] == 'elasticsearch' %}
```
Comment (zmerlynn) [inline cluster/saltbase/salt/top.sls]: Any reason not to lift the first condition up into line 59, and just make this an if/elif block? Just a minor nit

Comment (ArtfulCoder) [inline cluster/saltbase/salt/top.sls]: @zmerlynn 
Absolutely!

Updated PR.
PTAL

**Comment by ArtfulCoder** [comment]
Updated PR.
PTAL

**Comment by zmerlynn** [comment]
LGTM

**Pull request merged**